src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstance.cs

```diff
@@ -125,6 +131,19 @@ public VisualStudioInstance(Process hostProcess, DTE dte, ImmutableHashSet<strin
             CleanUp();
         }
 
+        private static string GetIpcClientChannelName(Process hostProcess)
+        {
+            var index = Interlocked.Increment(ref s_connectionIndex) - 1;
+            if (index == 0)
+            {
+                return $"IPC channel client for {hostProcess.Id}";
+            }
+            else
+            {
+                return $"IPC channel client for {hostProcess.Id} ({index})";
```

Comment:
Just curious, why can't we always append the index value to the name?

Reply:
No real reason I suppose, I just kept the current behavior for the base case